docs/DeveloperGuide.md

```diff
@@ -158,6 +158,18 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### View all feature
+
+#### Implementation
+
+The mechanism is similar to list feature in `AddressBook`. parser checks for `-all` flag and execute showing the entire list of students.
+
+The feature can return user to the whole list after user uses view -id/view -name function to see specific student. User has to add in all people before being able to use view all.
```

Comment:
"User has to add in all people before being able to use view all." 

Perhaps might be better phrased as "A list of students will only be displayed if there are at least 1 student added to TuteeTally"? Or even "If no students have been added to TuteeTally, view -all will return an empty list"